dev/snippets/lib/configuration.dart

```diff
@@ -67,8 +67,9 @@ class Configuration {
   /// dartdoc.
   Directory get templatesDirectory => Directory(path.join(configDirectory.path, 'templates'));
 
-  /// Gets the skeleton file to use for the given [SnippetType].
-  File getHtmlSkeletonFile(SnippetType type) {
-    return File(path.join(skeletonsDirectory.path, '${getEnumName(type)}.html'));
+  /// Gets the skeleton file to use for the given [SnippetType] and DartPad preference.
+  File getHtmlSkeletonFile(SnippetType type, bool useDartPad) {
```

Comment:
Make the bool argument named. It's confusing to have a bunch of `bool` arguments, because it just looks like `Foo(true, true, false)`, whereas naming the bool args is self-documenting: `Foo(delete: true, asRoot: true, askFirst: false)`.

```suggestion
  File getHtmlSkeletonFile(SnippetType type, {bool useDartPad = false}) {
```

Reply:
Good point. I renamed the param as well to bring it in line with the other "showDartPad" parameters used elsewhere.